taca_ngi_pipeline/deliver/deliver_castor.py

```diff
@@ -0,0 +1,170 @@
+""" 
+    Module for controlling deliveries os samples and projects to castor (THE castor!!!!)
+"""
+
+import paramiko
+import getpass
+import glob
+import time
+
+
+from deliver import *
+
+
+## cehcks if a file/folder exists in the remote location
+## http://stackoverflow.com/questions/850749/check-whether-a-path-exists-on-a-remote-host-using-paramiko
+def rexists(sftp, path):
+    """os.path.exists for paramiko's SCP object
+    """
+    try:
+        sftp.stat(path)
+    except IOError, e:
+        if 'No such file' in str(e):
+            return False
+        raise
+    else:
+        return True
+
+
+class CastorProjectDeliverer(ProjectDeliverer):
+    """ This object takes care of delivering project samples to castor's wharf.
+    """
+    def __init__(self, projectid=None, sampleid=None, **kwargs):
+        super(CastorProjectDeliverer, self).__init__(
+            projectid,
+            sampleid,
+            **kwargs)
+        # castor specific fields
```

Comment:
I'm just noting that these parameters are more or less project-specific. It's probably easier to pass them on the command line rather than parse them from the config